randovania/games/am2r/logic_database/header.txt

```diff
@@ -225,11 +225,15 @@ Templates
               Missiles ≥ 30 and Combat (Intermediate)
               Charge Beam and Missile-Less Metroid Fights (Advanced)
           Any of the following:
-              # Energy requirements
-              Normal Damage ≥ 199
+              # Energy Requirements
+              Combat (Advanced) or Normal Damage ≥ 299
```

Comment:
i wonder if this should  be expert instead of advanced. cause doing both min items + hitless is way harder

Reply:
Are these health values good, or should I bump them both up by 100
Zeta's trickless 399, Omega's 499?

Reply:
health values seem fine? sam had trouble fighting a zeta with 4 etanks on hard (299*1,5=448), and you take double damage there.

Reply:
figured since we bumped the super-less fights it might be good to get the health up a bit. But I'm not too concerned.